infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/Action.java

```diff
@@ -47,12 +51,14 @@ public void execute(@Nonnull CorfuRuntime runtime, int numRetry) {
                         getName(), x, e);
                 changeStatus(ActionStatus.ERROR);
                 runtime.invalidateLayout();
+                TimeUnit.MILLISECONDS.sleep(retryInterval);
```

Comment:
Invalidate will also wait if a layout server cannot be reached. There is a retry parameter in the api the ends up calling this. Why is it necessary to have the sleep here? Shouldn't the calling API just increase its number of retries?